definitions/ext-flex_host/definition.yml

```diff
@@ -0,0 +1,30 @@
+domain: EXT
+type: FLEX_HOST
+synthesis:
+  name: displayName
+  identifier: displayName
+  conditions:
+  - attribute: eventType
+    value: FlexHostSample
+  tags:
+    arch:
+      multiValue: false
+    coreCount:
+      multiValue: false
+    displayName:
+      multiValue: false
+    fullHostname:
+      multiValue: false
+    longHostname:
```

Comment:
wondering on the difference between fullHostname, longHostname & shortHostname do we really need them all?

Reply:
![image](https://user-images.githubusercontent.com/13043208/142386719-a528a156-228f-4a03-93e1-651a1d855ce3.png)
Have removed one of the hostname attributes, but it is to keep parity with how the infra agent does it.